src/VirtualClient/VirtualClient.Main/profiles/PERF-SPECJBB.json

```diff
@@ -28,20 +28,20 @@
             "Parameters": {
                 "Scenario": "InstallSPECjbbPackage",
                 "BlobContainer": "packages",
-                "BlobName": "specjbb.2015.0.0.zip",
+                "BlobName": "specjbb2015-1.04.zip",
```

Comment:
Just to make sure we uploaded to both virtualclientinternal and virtualclient storage